### ISSUE TITLE
Are there any plans on uploading a more recent version of pynipap to pypi?

### ISSUE THREAD

**Issue by chilicheech** [SpriteLink/NIPAP#1393]
The latest version there is 0.29.5 and the latest release is 0.32.5. Would it be possible to upload the latest version to pypi?

**Comment by garberg** [comment]
Yes, that can and should definitely be done! Thanks for pointing it out :)
